tensorflow/core/kernels/softmax_op_gpu.cu.cc

```diff
@@ -155,17 +161,17 @@ class SoftmaxOpGPU : public OpKernel {
                      context->allocate_temp(DataTypeToEnum<acc_type>::value,
                                             softmax_out->shape(), &sum_probs));
 
-      DoRowReduction<T, cub::Max, const T*>(
+      DoRowReduction<T, gpuprim::Max, const T*>(
           context, const_cast<T*>(max_logits.flat<T>().data()),
           reinterpret_cast<const T*>(logits_in_.flat<T>().data()), rows, cols);
 
       const int numThreads = 128;
       const int numBlocks = Eigen::divup(rows * cols, numThreads);
 
-      cub::CountingInputIterator<int> counting_iterator(0);
-      typedef cub::TransformInputIterator<acc_type,
-                                          SubtractAndExpFunctor<T, acc_type>,
-                                          cub::CountingInputIterator<int>>
+      gpuprim::CountingInputIterator<int> counting_iterator(0);
+      typedef gpuprim::TransformInputIterator<
```

Comment:
You are inheriting this, but we prefer 'using' these days.

Reply:
done